Tests/FluentAssertions.Specs/Primitives/StringAssertionSpecs.MatchRegex.cs

```diff
@@ -199,6 +199,159 @@ public void When_a_string_is_matched_against_an_empty_regex_it_should_throw_with
                 .WithParameterName("regularExpression");
         }
 
+        [Fact]
+        public void When_a_string_is_matched_against_a_regex_and_the_count_of_matches_fits_into_the_expected_it_passes()
```

Comment:
⛏️ You don't really need to repeat the fact that you're testing against a regex. That's clear from the context.

Reply:
hmm.. true